feat: add MacOS print options (#1259)
tauri-apps/wry: .changes/macos_print_margin_options.md

---
"wry": minor
---

> Add a new function to WebViewExtMacOS called print_with_options which
> allows the user to modify the margins that will be sent down to the AppKit print operation (NSPrintInfo).

Reviewer: ```suggestion
Add `WebViewExtMacOS::print_with_options` which allows to modify the margins that will be used on the print dialog.
```
